libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ExporterHelper.java

```diff
@@ -20,10 +22,20 @@
  */
 public interface ExporterHelper<T> {
 
-    public int getColumns();
+    static List<String> addSubheadersWithPrefixesAsNeeded(List<String> headers, List<String> subheaders, String prefix) {
+        List<String> prefixedSubheaders = subheaders
+                .stream()
+                .map(h -> headers.contains(h) ? prefix + h : h)
```

Comment:
Releases have substructures like clearingInformation, for example. If I were to prefix all subheaders, we'd get some really long headers like `release: clearing information: componentLicenseInformation`. I didn't like that prospect, so I decided to only prefix duplicate headers.